src/ert/config/parsing/_option_dict.py

```diff
@@ -1,11 +1,27 @@
 import logging
 from collections.abc import Sequence
 
-from .parsing import ConfigValidationError
+from .config_errors import ConfigValidationError
 
 logger = logging.getLogger(__name__)
 
 
+def parse_variable_options(
```

Comment:
Why is this function moved and renamed? Separate commit?

Reply:
Moved the change into separate commit. The function is moved as it will only be used in parsing.